File: week-02/day-3/double-items.py
# - Create an array variable named `ag`
#   with the following content: `[3, 4, 5, 6, 7]`
# - Double all the values in the array

ag = [3, 4, 5, 6, 7]
x = 0

while x < len(ag):
    ag[x] *= 2
    x += 1
print(ag)

File: week-07/day-3/double-items.js
'use strict';
// - Create an array variable named `ag` with the following content: `['Gin', 'Whiskey', 'Wine', 'Beer']`
// - Double all the strings in the array, use a built in array method instead of a loop
// It should print: ['GinGin', 'WhiskeyWhiskey', 'WineWine', 'BeerBeer']`

var ag = ['Gin', 'Whiskey', 'Wine', 'Beer'];

var doubleAg = ag.map(function(e, i){
    return e + e;
})
console.log(doubleAg);